setup.py

```diff
@@ -30,7 +30,7 @@ def get_all_extras() -> Dict:
 
     fetch_ledger_deps = ["fetchai-ledger-api==1.1.0"]
 
-    ethereum_ledger_deps = ["web3==5.2.2", "eth-account==0.4.0"]
+    ethereum_ledger_deps = ["web3==5.2.2", "ipfshttpclient==0.4.12", "eth-account==0.4.0"]
```

Comment:
It's not an ethereum ledger dependency; it's a general dependency...

Reply:
It's a sub-dependency of `web3==5.2.2`. In fact, `ipfshttpclient` was not used from the framework; only in `scripts/generate_ipfs_hashes.py`.

Reply:
Ah ok; I see now; makes sense!